python/tests/transformers/named_image_test.py

```diff
@@ -69,8 +69,9 @@ class NamedImageTransformerBaseTestCase(SparkDLTestCase):
     name = None
     # Allow subclasses to force number of partitions - a hack to avoid OOM issues
     numPartitionsOverride = None
-    featurizerCompareDigitsExact = 6
+    featurizerCompareDigitsExact = 5
```

Comment:
one easy way to make this global is by adding an `assertArrayAlmostEqual` method to `SparkDLTestCase` that calls `np.testing.test_arrays_almost_equal` and takes a default comparing precision, and using that instead of assertEquals. How do you feel about this?

Reply:
Good idea. It may needs to change all of the usage of `np.testing.test_arrays_almost_equal`. It might be better to change it on a different PR.